server.js

```diff
@@ -1,20 +1,16 @@
 const express = require("express");
-const path = require("path");
-const PORT = process.env.PORT || 3001;
+const PORT = process.env.PORT || 6003;
 const app = express();
 const keys = require("./config/keys");
 const mongoose = require("mongoose");
 
-// Serve up static assets (usually on heroku)
 if (process.env.NODE_ENV === "production") {
   app.use(express.static("client/build"));
 }
 
-// Send every request to the React app
-// Define any API routes before this runs
-app.get("*", (req, res) => {
-  res.sendFile(path.join(__dirname, "./client/build/index.html"));
-});
```

Comment:
we might want to leave this as `/`, I don't think the client will load without it